packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx

```diff
@@ -207,6 +227,16 @@ export const FeedTipTile = () => {
     getUsers(state, { ids: tipToDisplay ? tipperIds : [] })
   )
 
+  const [hasShortButtonChanged, setHasShortButtonChanged] = useState(false)
+  const containerRef: MutableRefObject<HTMLDivElement | null> = useRef(null)
+  const containerLeftContentsRef: MutableRefObject<HTMLDivElement | null> =
+    useRef(null)
+  const sendTipRef: MutableRefObject<HTMLDivElement | null> = useRef(null)
+  const firstLoadRef = useRef(true)
+  const maxTipButtonWidth = useRef(0)
+  const useShortButtonFormat = useRef(false)
```

Comment:
I think this can probably be a state instead of ref? What's the reasoning for a ref here?